config/locales/account/en.yml

```diff
@@ -109,4 +109,5 @@ en:
     security:
       link: Learn more at the Help Center
       text: Your profile information is locked for your security.
+    terms: I agree to login.gov Terms of Use and the Security and Privacy Practices.
```

Comment:
Was there consensus on whether we're calling this "Rules of Use" or "Terms of Use" ? We use "Terms" here, and "Rules" in #5040.

Reply:
I posted the discrepancy in slack and they said roll with it